angrmanagement/data/object_container.py

```diff
@@ -29,6 +29,12 @@ def am_event(self, **kwargs):
 
 
 class ObjectContainer(EventSentinel):
+    """
```

Comment:
This docstring looks like it is not supposed to be part of this PR

Reply:
It was for ensuring CI passed linting and I have no issues adding documentation to code anywhere.